modules/ROOT/pages/application-deployment.adoc

```diff
@@ -18,7 +22,14 @@ Where `foo` is a Mule app at `$MULE_HOME/apps/foo`.
 
 From this moment, Mule checks every three seconds for the `$MULE_HOME/apps/foo/mule-config.xml` updates. You can update the application `jar` contents and modify this file so Mule reloads the config and class modifications.
 
-Hot deployment doesn’t work when using the `mule.deploy.applications` property.  Only applications in the `apps` directory are deployed, new or updated applications don't start until Mule restarts.
+Start Mule by specifying multiple apps to run, for example:
```

Comment:
Start Mule by specifying multiple apps to run, for example:
-->
Start Mule and specify multiple apps to deploy, for example:

Reply:
Updated.